frontend/src/app/testResults/viewResults/actionMenuModals/TestResultDetailsModal.tsx

```diff
@@ -199,11 +184,11 @@ export const DetachedTestResultDetailsModal = ({
           )}
           <DetailsRow
             label="Pregnant?"
-            value={pregnancy && pregnancyMap[pregnancy]}
+            value={pregnancy && pregnancyMap[pregnancy as keyof Pregnancy]}
             removed={removed}
             aria-describedby="result-detail-title"
           />
-          {isHIVResult && (
+          {(isHIVResult || isSyphilisResult) && (
```

Comment:
⚠️ There will come a day where we need to update how we handle the display logic for which diseases should show which AOE options, but today is not that day (prioritizing getting syphilis feature out)